Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/ui/onboarding/validationrules/info/ValidationRulesInfoFragment.kt

```diff
@@ -0,0 +1,30 @@
+package de.rki.coronawarnapp.covidcertificate.ui.onboarding.validationrules.info
```

Comment:
we should move it to `de.rki.coronawarnapp.covidcertificate.validation.ui.datetimeinfo` , it is not part of the onboarding